config/configgrpc/configgrpc.go

```diff
@@ -320,6 +320,22 @@ func validateBalancerName(balancerName string) bool {
 	return balancer.Get(balancerName) != nil
 }
 
+func (gss *ServerConfig) Validate() error {
+	if gss.MaxRecvMsgSizeMiB*1024*1024 < 0 {
+		return fmt.Errorf("invalid max_recv_msg_size_mib value: %d", gss.MaxRecvMsgSizeMiB)
```

Comment:
Cant he error messages include the max valid value?

Reply:
Done